.github/workflows/twister-prep.yaml

```
@@ -0,0 +1,150 @@
+name: Prep
```

```suggestion
name: Twister Prep
```

to make it more clear what it is in the workflow list ...